website/routes.py

```diff
@@ -882,6 +882,24 @@ def make_url_map(app):
             project_views.register.node_registration_embargo_disapprove,
             OsfWebRenderer('error.mako', trust=False)
         ),
+        Rule(
```

Comment:
(in response JeffSpies) perhaps the easiest way to implement this would be JWT or equivalent? It's conceivable we want the ability to frontend decode tokens in the future. There is some challenge in guaranteeing uniqueness, but we could always append/prepend unique strings to tokens. 

At any rate such a change would be transparent to users and could happen after the initial deployment of this PR.